great_ape_safe/ape_api/__init__.py

```diff
@@ -1,11 +1,94 @@
-import glob
-import os
+from .aave import Aave
+from .anyswap import Anyswap
+from .aura import Aura
+from .badger import Badger
+from .balancer import Balancer
+from .bunni import Bunni
+from .chainlink import Chainlink
+from .compound import Compound
+from .convex import Convex
+from .cow import Cow
+from .curve_v2 import CurveV2
+from .curve import Curve
+from .euler import Euler
+from .maker import Maker
+from .opolis import Opolis
+from .pancakeswap_v2 import PancakeswapV2
+from .rari import Rari
+from .snapshot import Snapshot
+from .solidly import Solidly
+from .spookyswap import Spookyswap
+from .sushi import Sushi
+from .uni_v2 import UniV2
+from .uni_v3 import UniV3
 
 
-files = glob.glob("great_ape_safe/ape_api/[!_]*.py")
-ape_apis = {}
-for file in files:
-    module_path = os.path.splitext(file)[0].replace("/", ".")
-    module = __import__(module_path, fromlist=[module_path.capitalize()])
-    class_name = module_path.split(".")[-1].title().replace("_", "")
-    ape_apis[module_path.split(".")[-1]] = getattr(module, class_name)
+class ApeApis:
+    def init_aave(self):
+        self.aave = Aave(self)
+
+    def init_anyswap(self):
+        self.anyswap = Anyswap(self)
+
+    def init_aura(self):
+        self.aura = Aura(self)
+
+    def init_badger(self):
+        self.badger = Badger(self)
+
+    def init_balancer(self):
+        self.balancer = Balancer(self)
+
+    def init_bunni(self):
+        self.bunni = Bunni(self)
+
+    def init_chainlink(self):
+        self.chainlink = Chainlink(self)
+
+    def init_compound(self):
+        self.compound = Compound(self)
+
+    def init_convex(self):
+        self.convex = Convex(self)
+
+    def init_cow(self):
```

Comment:
for the cases like cow or bunni that may be expecting another param this may not work, no?

e.g : `prod` or `bunni_token_addr`

Reply:
good catch 92ab8c4